src/sentry/api/serializers/rest_framework/groupsearchview.py

```diff
@@ -1,11 +1,15 @@
 from typing import Any, NotRequired, TypedDict
```

Comment:
doesn't have to be now, but would be nice if we could move this serializer to a folder in the issues domain (so `src/sentry/issues/serializers/groupsearchview.py`.